### PR TITLE
Enhanced test case for Noise ability

### PR DESCRIPTION
**Pull request by kevinpowe**
Incorporating suggestions from previous pull request- now uses Clone Chip on Corp's turn.

### CODE REVIEW

File: src/clj/test/cards/identities.clj [hunk 1]
```diff
@@ -652,20 +652,45 @@
...
   (do-game
     (new-game
-      (default-corp [(qty "Hedge Fund" 3) (qty "Restructure" 3) (qty "PAD Campaign" 3)])
-      (make-deck "Noise: Hacker Extraordinaire" [(qty "Datasucker" 1) (qty "Easy Mark" 2)]))
+      (default-corp [(qty "Hedge Fund" 3) (qty "Restructure" 3) (qty "PAD Campaign" 3) (qty "Beanstalk Royalties" 2)])
+      (make-deck "Noise: Hacker Extraordinaire" [(qty "Datasucker" 1) (qty "Cache" 1) (qty "Sure Gamble" 1) (qty "Clone Chip" 2) (qty "Sharpshooter" 2)]))
+    (starting-hand state :runner ["Datasucker" "Sure Gamble" "Clone Chip" "Clone Chip" "Cache"])
     (is (= 6 (count (:hand (get-corp)))) "Corp should start with 6 cards in hand")
-    (is (= 3 (count (:deck (get-corp)))) "Corp deck should contain 3 cards")
+    (is (= 5 (count (:deck (get-corp)))) "Corp deck should contain 3 cards")
```
Comment (erbridge) [inline src/clj/test/cards/identities.clj]: The message should say `5 cards` now.

**Comment by kevinpowe** [comment]
Thanks @erbridge! I missed a push from my work area before creating the pull request. Still wrapping my head around the git workflow. Feel like a moron for missing this!

**Comment by mtgred** [comment]
👍

**Comment by kevinpowe** [comment]
Thanks @mtgred! Got buried under work, just got back to this this morning. Thank you for the help!

**Pull request merged**